views/play.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <link rel="stylesheet" href="/style.css">
```

Comment:
Nice use of external styling for your HTML document Jaabir. This improves the readability of your code!